source/common/quic/envoy_quic_client_stream.cc

```diff
@@ -194,6 +202,11 @@ void EnvoyQuicClientStream::OnInitialHeadersComplete(bool fin, size_t frame_len,
     set_headers_decompressed(false);
   }
 
+  // Transform Response from H3 to H1
```

Comment:
Ditto here about explaining why we're doing this (which is that we received the HTTP/3 form and need to convert to the canonical Envoy representation)